services/ui-src/src/components/banners/Banner.tsx

```diff
@@ -0,0 +1,55 @@
+// components
+import {
+  Alert,
+  AlertDescription,
+  AlertIcon,
+  AlertTitle,
+  Flex,
+} from "@chakra-ui/react";
+
+import { BannerTypes } from "utils/types/types";
+
+// Make alert banner and pass in status and color, text,
+export const Banner = ({
+  status,
+  bgColor,
+  accentColor,
+  title,
+  description,
+}: Props) => {
+  return (
+    <Alert
+      sx={sx.root}
+      status={status}
+      variant="left-accent"
+      bg={bgColor}
+      borderInlineStartColor={accentColor}
+      data-testid="banner"
```

Comment:
If we're going to have multiples of these in the future we might want to make `data-testid` a passable prop so we can test that different ones show up on the same page. Not super likely but a thought. I'm just writing all my thoughts. Idk. Hope you enjoy this journey through my thoughts late on a Friday.